docs/function-format.md

```diff
@@ -58,42 +62,44 @@ The header keys and values would be populated with information about the functio
 
 Pros:
 
-* Streamable
+* Supports streaming
 * Common format
 
 Cons:
 
 * Requires a parsing library or fair amount of code to parse headers properly
 * Double parsing - headers + body (if body is to be parsed, such as json)
 
-#### JSON I/O Format (not implemented)
+#### JSON Input Format
 
 `--format json`
 
-The idea here is to keep the HTTP base structure, but make it a bit easier to parse by making the `request line` and `headers` a JSON struct.
-Eg:
+An easy to parse JSON structure.
 
-```
+```json
 {
-  "request_url":"http://....",
-  "params": {
-    "blog_name": "yeezy"
+  "body": {
+    "some": "input"
   }
 }
-BLANK LINE
-BODY
 ```
 
 Pros:
 
 * Streamable
-* Easy to parse headers
+* Easy to parse
 
 Cons:
 
-* New, unknown format
+* ???
```

Comment:
not streamable, actually, as we have defined it

Reply:
Sure, maybe say hot capable or something then.